llvm-external-projects/iree-dialects/lib/Dialect/VectorExt/IR/VectorExtDialect.cpp

```diff
@@ -109,17 +109,41 @@ VectorLayoutInterface LayoutAttr::permute(ArrayRef<int64_t> permutation) const {
   return LayoutAttr::get(getContext(), newLayouts);
 }
 
-SmallVector<int64_t> LayoutAttr::getDistributedShape(VectorType) const {
-  LayoutDimension labels[] = {LayoutDimension::BATCHX, LayoutDimension::BATCHY,
-                              LayoutDimension::VECTORX};
+SmallVector<int64_t> LayoutAttr::getDistributedShape() const {
+  SmallVector<LayoutDimension> labels{
```

Comment:
Can you add a comment here that this is the order in which the shape is distributed?

Reply:
Sure, will add.